precompiles/EcPairing.yul

```diff
@@ -1633,10 +1608,6 @@ object "EcPairing" {
                 g2_y0 := intoMontgomeryForm(g2_y0)
                 g2_y1 := intoMontgomeryForm(g2_y1)
 
-                if iszero(g2IsInSubGroup(g2_x0,g2_x1, g2_y0, g2_y1, MONTGOMERY_ONE(), 0)) {
```

Comment:
This is not correct.

In the geth implementation mentioned the `G2` actually uses a `twistPoint`: https://github.com/ethereum/go-ethereum/blob/master/crypto/bn256/google/bn256.go#L166.
And its `IsOnCurve()` method has the subgroup check: https://github.com/ethereum/go-ethereum/blob/master/crypto/bn256/google/twist.go#L84